commit d30409f6d173041be7730c84f4c7784dc159199f
Author: Adam Jackson <ajax@redhat.com>
Date:   Mon Feb 15 17:19:48 2016 -0500

    demos: Read from installed paths

diff --git a/demos/cube.c b/demos/cube.c
index f337e0f..68a2215 100644
--- a/demos/cube.c
+++ b/demos/cube.c
@@ -115,7 +115,7 @@ struct texture_object {
     int32_t tex_width, tex_height;
 };
 
-static char *tex_files[] = {"lunarg.ppm"};
+static char *tex_files[] = {"/usr/share/vulkan/cube.ppm"};
 
 struct vkcube_vs_uniform {
     // Must start with MVP
@@ -1478,7 +1478,7 @@ static VkShaderModule demo_prepare_vs(struct demo *demo) {
     void *vertShaderCode;
     size_t size;
 
-    vertShaderCode = demo_read_spv("cube-vert.spv", &size);
+    vertShaderCode = demo_read_spv("/usr/share/vulkan/cube-vert.spv", &size);
 
     demo->vert_shader_module =
         demo_prepare_shader_module(demo, vertShaderCode, size);
@@ -1492,7 +1492,7 @@ static VkShaderModule demo_prepare_fs(struct demo *demo) {
     void *fragShaderCode;
     size_t size;
 
-    fragShaderCode = demo_read_spv("cube-frag.spv", &size);
+    fragShaderCode = demo_read_spv("/usr/share/vulkan/cube-frag.spv", &size);
 
     demo->frag_shader_module =
         demo_prepare_shader_module(demo, fragShaderCode, size);
diff --git a/demos/tri.c b/demos/tri.c
index 5910c37..87f2cb7 100644
--- a/demos/tri.c
+++ b/demos/tri.c
@@ -1229,7 +1229,7 @@ static VkShaderModule demo_prepare_vs(struct demo *demo) {
     void *vertShaderCode;
     size_t size;
 
-    vertShaderCode = demo_read_spv("tri-vert.spv", &size);
+    vertShaderCode = demo_read_spv("/usr/share/vulkan/tri-vert.spv", &size);
 
     demo->vert_shader_module =
         demo_prepare_shader_module(demo, vertShaderCode, size);
@@ -1243,7 +1243,7 @@ static VkShaderModule demo_prepare_fs(struct demo *demo) {
     void *fragShaderCode;
     size_t size;
 
-    fragShaderCode = demo_read_spv("tri-frag.spv", &size);
+    fragShaderCode = demo_read_spv("/usr/share/vulkan/tri-frag.spv", &size);
 
     demo->frag_shader_module =
         demo_prepare_shader_module(demo, fragShaderCode, size);
