--- a/demos/CMakeLists.txt
+++ b/demos/CMakeLists.txt
@@ -50,24 +50,6 @@ if(WIN32)
    file(COPY cube.vcxproj.user DESTINATION ${CMAKE_BINARY_DIR}/demos)
    file(COPY tri.vcxproj.user DESTINATION ${CMAKE_BINARY_DIR}/demos)
    file(COPY vulkaninfo.vcxproj.user DESTINATION ${CMAKE_BINARY_DIR}/demos)
-else()
-    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/demos/tri-vert.spv
-       COMMAND ${GLSLANG_PREFIX}/build/install/bin/glslangValidator -s -V -o tri-vert.spv ${PROJECT_SOURCE_DIR}/demos/tri.vert 
-       DEPENDS tri.vert ${GLSLANG_PREFIX}/build/install/bin/glslangValidator
-       )
-    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/demos/tri-frag.spv
-       COMMAND ${GLSLANG_PREFIX}/build/install/bin/glslangValidator -s -V -o tri-frag.spv ${PROJECT_SOURCE_DIR}/demos/tri.frag
-       DEPENDS tri.frag ${GLSLANG_PREFIX}/build/install/bin/glslangValidator
-       )
-    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/demos/cube-vert.spv
-       COMMAND ${GLSLANG_PREFIX}/build/install/bin/glslangValidator -s -V -o cube-vert.spv ${PROJECT_SOURCE_DIR}/demos/cube.vert
-       DEPENDS cube.vert ${GLSLANG_PREFIX}/build/install/bin/glslangValidator
-       )
-
-    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/demos/cube-frag.spv
-       COMMAND ${GLSLANG_PREFIX}/build/install/bin/glslangValidator -s -V -o cube-frag.spv ${PROJECT_SOURCE_DIR}/demos/cube.frag
-       DEPENDS cube.frag ${GLSLANG_PREFIX}/build/install/bin/glslangValidator
-       )
 endif()
 
 if(NOT WIN32)
@@ -96,14 +78,14 @@ target_link_libraries(vulkaninfo ${LIBRA
 install(TARGETS vulkaninfo RUNTIME DESTINATION /usr/bin)
 
 if(UNIX)
-    add_executable(tri tri.c ${CMAKE_BINARY_DIR}/demos/tri-vert.spv ${CMAKE_BINARY_DIR}/demos/tri-frag.spv)
+    add_executable(tri tri.c ${CMAKE_SOURCE_DIR}/debian/local/tri-vert.spv ${CMAKE_SOURCE_DIR}//debian/local/tri-frag.spv)
 else()
     add_executable(tri WIN32 tri.c ${CMAKE_BINARY_DIR}/demos/tri-vert.spv ${CMAKE_BINARY_DIR}/demos/tri-frag.spv)
 endif()
 target_link_libraries(tri ${LIBRARIES})
 
 if(NOT WIN32)
-    add_executable(cube cube.c ${CMAKE_BINARY_DIR}/demos/cube-vert.spv ${CMAKE_BINARY_DIR}/demos/cube-frag.spv)
+    add_executable(cube cube.c ${CMAKE_SOURCE_DIR}/debian/local/cube-vert.spv ${CMAKE_SOURCE_DIR}//debian/local/cube-frag.spv)
     target_link_libraries(cube ${LIBRARIES})
 else()
     if (CMAKE_CL_64)
