--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,8 +31,8 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux
 #   add_definitions(-DVK_USE_PLATFORM_MIR_KHR)
 #   set(DisplayServer Mir)
 
-#   add_definitions(-DVK_USEPLATFORM_WAYLAND_KHR)
-#   set(DisplayServer Wayland)
+    add_definitions(-DVK_USEPLATFORM_WAYLAND_KHR)
+    set(DisplayServer Wayland)
 
 else()
     message(FATAL_ERROR "Unsupported Platform!")
@@ -65,10 +65,10 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_C
 endif()
 
 option(BUILD_LOADER "Build loader" ON)
-option(BUILD_TESTS "Build tests" ON)
+option(BUILD_TESTS "Build tests" OFF)
 option(BUILD_LAYERS "Build layers" ON)
 option(BUILD_DEMOS "Build demos" ON)
-option(BUILD_VKJSON "Build vkjson" ON)
+option(BUILD_VKJSON "Build vkjson" OFF)
 
 if (BUILD_TESTS)
     # Hard code our glslang path for now
