--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,10 +65,10 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_C
 endif()
 
 option(BUILD_LOADER "Build loader" ON)
-option(BUILD_TESTS "Build tests" ON)
+option(BUILD_TESTS "Build tests" OFF)
 option(BUILD_LAYERS "Build layers" ON)
 option(BUILD_DEMOS "Build demos" ON)
-option(BUILD_VKJSON "Build vkjson" ON)
+option(BUILD_VKJSON "Build vkjson" OFF)
 
 if (BUILD_TESTS)
     # Hard code our glslang path for now
