Description: use -mxgot flag on mips64
Author: Dejan Latinovic <Dejan.Latinovic@imgtec.com>

--- a/layers/CMakeLists.txt
+++ b/layers/CMakeLists.txt
@@ -104,6 +104,12 @@ else()
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wpointer-arith")
 endif()
 
+if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "mips64")
+    if("${CMAKE_SIZEOF_VOID_P}" EQUAL "8")
+        set_source_files_properties(core_validation.cpp PROPERTIES COMPILE_FLAGS -mxgot)
+    endif()
+endif()
+
 add_custom_command(OUTPUT vk_dispatch_table_helper.h
     COMMAND ${PYTHON_CMD} ${PROJECT_SOURCE_DIR}/vk-generate.py ${DisplayServer} dispatch-table-ops layer > vk_dispatch_table_helper.h
     DEPENDS ${PROJECT_SOURCE_DIR}/vk-generate.py ${PROJECT_SOURCE_DIR}/vulkan.py)
